code/drasil-database/lib/Database/Drasil/ChunkDB.hs

```diff
@@ -5,28 +5,7 @@
 -- in Drasil' tutorial found on the wiki:
 -- https://github.com/JacquesCarette/Drasil/wiki/Creating-Your-Project-in-Drasil
 
-module Database.Drasil.ChunkDB (
-  -- * Types
-  -- ** 'ChunkDB'
-  -- | Main database type
-  ChunkDB(CDB, symbolTable, termTable, defTable),
-  -- ** Maps
-  -- | Exported for external use.
-  RefbyMap, TraceMap, UMap,
-  -- * Functions
-  -- ** Constructors
-  cdb, idMap, termMap, conceptMap, traceMap, generateRefbyMap, -- idMap, termMap for docLang
-  -- ** Lookup Functions
-  asOrderedList, collectUnits,
-  termResolve, defResolve, symbResolve,
-  traceLookup, refbyLookup,
-  datadefnLookup, insmodelLookup, gendefLookup, theoryModelLookup,
-  conceptinsLookup, sectionLookup, labelledconLookup, refResolve,
-  -- ** Lenses
-  unitTable, traceTable, refbyTable,
-  dataDefnTable, insmodelTable, gendefTable, theoryModelTable,
-  conceptinsTable, sectionTable, labelledcontentTable, refTable
-  ) where
+module Database.Drasil.ChunkDB where
 
```

Comment:
What's the rationale behind removing list of what is exported? If pretty much everything is exported and there's nothing that should be explicitly hidden, then this seems fine to me lol

Reply:
They are indeed tedious to maintain, but I think they are a solid piece of documentation.

Very long lists of exports are a code smell (exception: drasil-data and utilities).

Reply:
Reverted in 934dfdd